docs/data/routes/docs/nextjs/getting-started-nextjs/walkthrough-jsscreate/en.md

```diff
@@ -75,5 +75,5 @@ Sitecore requires Windows, but the instance can be on a virtual machine or a rem
 To connect your application to Sitecore, you must:
 
 1. [Setup JSS Server Components](/docs/client-frameworks/getting-started/jss-server-install).
-2. [Deploy the application to a Sitecore environment](/docs/client-frameworks/getting-started/app-deployment).
+2. Run `jss deploy items`.
```

Comment:
Much of that linked page is actually necessary. Not sure about the best way to communicate that (I'll leave it up to you :), but here's what is needed for Next.js:

**Configuring the Sitecore server to receive the JSS app**

- [NO] Step 1. Configure your app's site and host name
- [YES] Step 2. API Key
- [YES] Step 3. Verification

**Connecting the JSS app to Sitecore**

- [YES] Step 1: Setup connection information
- [YES] Step 2: Config deployment
- [YES(ISH)] Step 3: App Deployment
		This is where we want to callout that _only_ `jss deploy items` is required
- [NO] Step 4: Verification

Reply:
Awesome input @ambrauer. Thank you for the clarity.